x-pack/plugins/features/server/oss_features.ts

```diff
@@ -279,6 +291,9 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
             },
           ],
         },
+        ...(includeReporting
+          ? [reportingFeatures.downloadCsvDashboard, reportingFeatures.generateScreenshotDashboard]
```

Comment:
We can combine these two privileges into a single "sub feature privilege group" to streamline the UI a bit:

![image](https://user-images.githubusercontent.com/3493255/114909144-d10b0300-9dea-11eb-8dfb-1bccbbc4d47d.png)


Something like:

```ts
dashboardReporting: {
    name: i18n.translate('xpack.features.ossFeatures.reporting.dashboardReportingFeatureName', {
      defaultMessage: 'Reporting',
    }),
    privilegeGroups: [
      {
        groupType: 'independent',
        privileges: [
          {
            id: 'download_csv_dashboard_panel',
            name: i18n.translate('xpack.features.ossFeatures.reporting.dashboardDownloadCSV', {
              defaultMessage: 'Generate CSV Reports',
            }),
            includeIn: 'all',
            savedObject: { all: [], read: [] },
            management: { insightsAndAlerting: ['reporting'] },
            api: ['downloadCsv'],
            ui: ['downloadCsv'],
          },
          {
            id: 'generate_screenshot_report',
            name: i18n.translate(
              'xpack.features.ossFeatures.reporting.dashboardGenerateScreenshot',
              {
                defaultMessage: 'Generate PDF or PNG reports',
              }
            ),
            includeIn: 'all',
            minimumLicense: 'platinum' as 'platinum',
            savedObject: { all: [], read: [] },
            management: { insightsAndAlerting: ['reporting'] },
            api: ['generateReport'],
            ui: ['generateScreenshot'],
          },
        ],
      },
    ],
  },
```

Reply:
c293f110e97